components/core/View.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from 'react';
+
+import {View as RNView, ViewProps as RNViewProps, ViewStyle as RNViewStyle} from 'react-native';
+
+import {colorLookup} from 'theme';
+
+class ViewStyleProps {
```

Comment:
This is declared in a class so that we can instantiate it and get the list of keys into an array. You can't do that with an interface. Lord knows I tried 😄